DemoServer/Controllers/Demos/CompareExchange/IndexCompareExchange/metadata.json

```diff
@@ -0,0 +1,161 @@
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/client-api/operations/compare-exchange/overview"
+        },
+        {
+            "Title": "Index Compare-Exchange values",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/Csharp/indexes/indexing-compare-exchange-values#querying-the-index"
+        },
+        {
+            "Title": "Distributed Compare-Exchange operations",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/2-zero-to-ravendb#distributed-compare-exchange-operations-with-ravendb"
+        },
+        {
+            "Title": "Atomic Compare-Exchange operations",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/6-ravendb-clusters#distributed-atomic-compare-exchange-operations"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a static index",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Create a static Map index class - inherit from `AbstractIndexCreationTask`.",
+                "",
+                "* The generic params:<br>",
+                "    * _Product_ - indicates that the source collection for the index will be the 'Products' collection<br>",
+                "    * _Products_byUnitsInStock.IndexEntry_ - is the shape of the index entries (see next step)"
+            ],
+            "Assets": [
+                {
+                    "Title": "Static Indexes Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/static-indexes/static-indexes-overview"
+                },
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The index Entry class",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* The `IndexEntry` class defines the index-fields per entry in the index.<br>",
+                "  These fields will be indexed and can be queried on.",
+                "",
+                "* Field _UnitsInStock_ will be configured to have CmpXchg Values (see next step)."
```

Comment:
maybe "to hold CmpXchg" is slightly more precise wording?

Reply:
done